src/main/java/util/Ui.java

```diff
@@ -0,0 +1,125 @@
+package util;
+
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user
+ * */
+public class Ui {
+    /** Constants **/
+    private final String outputBreaker = ">>> ";
+    private final String lineBreaker = "--.--.--.--.--.--.--.--.--.--.--." +
```

Comment:
Perhaps consider naming it just `lineBreak`? Naming it `lineBreaker` makes it seem like it has a function.

Reply:
That is a good point, thanks!